slack/src/faceswapbot/faceswapbot.go

```diff
@@ -96,7 +96,8 @@ func main() {
 							ev.Channel,
 						))
 					}
-				} else if strings.Contains(strings.ToLower(ev.Text), "help") {
+				} else if strings.Contains(strings.ToLower(ev.Text), "help") ||
+						  strings.Contains(strings.ToLower(ev.Text), "hjalp") {
```

Comment:
This needs to be changed to match the implementation in https://github.com/digiaonline/cuddly-potato/blob/master/slack/src/faceswapbot/faceswapbot.go#L86-L88

i.e.
```go
} else if strings.HasSuffix(strings.ToUpper(strings.TrimSpace(ev.Text)), "help") ||
    strings.HasSuffix(strings.ToUpper(strings.TrimSpace(ev.Text)), "hjalp") {
```

forgot to change this here when I changed the `bomb` and `success` ifs. Using `Contains` will match _help_ regardless where the text is -> upload an image with the name "thelp.jpg" will trigger the help. By accident the help wouldn't be triggered in this case as the file upload "if" is done before the help section in https://github.com/digiaonline/cuddly-potato/blob/master/slack/src/faceswapbot/faceswapbot.go#L80 but you get the point.

Reply:
Updated!